bedrock/mozorg/templates/mozorg/home/includes/m24/donate.html

```diff
@@ -0,0 +1,21 @@
+{#
+ This Source Code Form is subject to the terms of the Mozilla Public
+ License, v. 2.0. If a copy of the MPL was not distributed with this
+ file, You can obtain one at https://mozilla.org/MPL/2.0/.
+#}
+
+<div class="m24-t-dark" id="donate">
+  <div class="m24-c-donate">
+    <h2 class="m24-c-donate-title">Donate to Mozilla’s non-profit</h2>
+    <div class="m24-c-donate-media"><img src="{{ static('img/m24/todo-27-1.svg') }}" alt="" height="253" width="680"></div>
+    <div class="m24-c-donate-info">
+      <p class="m24-c-donate-info-title">Where do your dollars go?</p>
+      <ul class="mzp-u-list-styled">
+        <li>Funding new AI privacy technologies</li>
+        <li>Building innovation labs</li>
+        <li>Sponsoring incoming 2025 Fellows</li>
+      </ul>
+    </div>
+    <p class="m24-c-donate-cta"><a href="#TODO" class="m24-c-cta" data-cta-text="Donate">Donate</a></p>
```

Comment:
**Nit:** This CTA should be left aligned on mobile:

<kbd>![Screenshot 2024-09-10 at 6 26 05 PM](https://github.com/user-attachments/assets/aeb976e3-811f-4aea-a1d9-52e5f0ec26dd)</kbd>